crowdin.yml

```diff
@@ -1,29 +1,28 @@
 #
 # Your Crowdin credentials
 #
-"project_id": "420172"                    # Crowdin project Settings > API & Webhooks > API > Project Id
-"api_token_env": "CROWDIN_PERSONAL_TOKEN" # Low priority: superseded by user-specific credentials file ~/.crowdin.yaml
-"base_path": "."
-"base_url": "https://api.crowdin.com"
+'project_id': '415116' # Crowdin project Settings > API & Webhooks > API > Project Id
```

Comment:
Let me double check for the project ID

Reply:
Allright!!

Reply:
We can keep using 420172 @NguyenTranHoangSym

Reply:
So that was the original. Can you also take a look if we have any keys tranlsated already there